sros/docker/launch.py

```diff
@@ -251,6 +251,20 @@ def line_card_config(chassis, card, mda, integrated=False, card_type=None):
             }
         ],
     },
+    "vsr-i": {
+        "deployment_model": "integrated",
+        "min_ram": 8,  # minimum RAM requirements
+        "max_nics": 20,
+        "timos_line": "chassis=VSR-I slot=A card=iom-v mda/1=m20-v mda/2=isa-tunnel-v",
+        "card_config": """/configure card 1 card-type iom-v
+        /configure card 1 mda 1 mda-type m20-v
+        /configure card 1 mda 2 mda-type isa-tunnel-v
+        """,
```

Comment:
Hi @mairklone 
would you be able to adapt this piece to using the line_card_config function
https://github.com/hellt/vrnetlab/blob/master/sros/docker/launch.py#L246

Reply:
Hi @hellt 
the only thing that I am seeing is that the function so far only allows me to add one mda:
https://github.com/hellt/vrnetlab/blob/0815321fab3d35dfabe4ecf25d7c91dbc8013c7b/sros/docker/launch.py#L49
In my humble opinion for this type of chassis being functional both mdas need to be provision, what do you think?

Reply:
ah, okay, we will have to enhance it then, but not in this PR
this looks good, thanks